README.md

```diff
@@ -1,61 +1,73 @@
+
+It was originally developed by [Kozec](https://github.com/kozec), then maintained in <https://github.com/syncthing-gtk/syncthing-gtk>.
 
 [![screenshot1](http://i.imgur.com/N36wmBM.png)](http://i.imgur.com/eX250tQ.png) [![screenshot2](http://i.imgur.com/43mmnC7.png)](http://i.imgur.com/RTRgRdC.png) [![screenshot3](http://i.imgur.com/KDBYekd.png)](http://i.imgur.com/OZ4xEeH.jpg)
 
-##### Supported Syncthing features
+## Supported Syncthing features
+
 - Everything what WebUI can display
 - Adding / editing / deleting nodes
 - Adding / editing / deleting repositories
 - Restart / shutdown server
 - Editing daemon settings
 
-##### Additional features
+## Additional features
+
 - First run wizard for initial configuration
 - Running Syncthing daemon in background
 - Half-automatic setup for new nodes and repositories
 - Nautilus (a.k.a. Files), Nemo and Caja integration
 - Desktop notifications
 
-##### Like what I'm doing?
-[![Help me become filthy rich on Liberapay](https://img.shields.io/badge/Help%20me%20become%20filthy%20rich%20on-Liberapay-yellow.svg)](https://liberapay.com/kozec) <sup>or</sup> [![donate anything with PayPal](https://img.shields.io/badge/donate_anything_with-Paypal-blue.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=77DQD3L9K8RPU&lc=SK&item_name=kozec&item_number=scc&currency_code=EUR&bn=PP%2dDonationsBF%3abtn_donate_LG%2egif%3aNonHosted)
-
-##### Packages
-- Ubuntu, Debian, deb-based distros: in [OpenSUSE Build Service](http://software.opensuse.org/download.html?project=home%3Akozec&package=syncthing-gtk).
-- Arch Linux: In [[community] repository](https://www.archlinux.org/packages/community/any/syncthing-gtk/)
-- Fedora: search for `syncthing-gtk` in Software Center
-- SUSE (and other rpm-based distros): in [OpenSUSE Build Service](http://software.opensuse.org/download.html?project=home%3Akozec&package=syncthing-gtk).
-- Flatpak (distro-agnostic): in [Flathub](https://flathub.org/apps/details/me.kozec.syncthingtk)
-- Windows: Get [latest installer from here](https://github.com/kozec/syncthing-gui/releases/latest), or use [the Chocolatey package](https://chocolatey.org/packages/syncthing-gtk).
-- Or, in worst case scenario, download [latest tarball](https://github.com/kozec/syncthing-gui/releases/latest), extract it and run syncthing-gtk.py.
-
-##### Dependencies
-- python 2.7, GTK 3.8 or newer and [PyGObject](https://live.gnome.org/PyGObject)
-- [python-gi-cairo](https://packages.debian.org/sid/python-gi-cairo),
-[gir1.2-notify](https://packages.debian.org/sid/gir1.2-notify-0.7) and [gir1.2-rsvg](https://packages.debian.org/sid/gir1.2-rsvg-2.0) on debian based distros (included in PyGObject elsewhere)
-- [python-dateutil](http://labix.org/python-dateutil) (Python 2 version)
+## How to install
+
+### On most Linux distributions
+
+Packages are available on most linux distributions as `syncthing-gtk`.
+
+It's available as Flatpak too [on Flathub](https://flathub.org/apps/details/me.kozec.syncthingtk)
```

Comment:
This is not true anymore — that version is not getting updated.

Reply:
Indeed. Also it seems there's a typo.